dev/ant_build/olt_imports.xml

```diff
@@ -9,25 +9,25 @@
  *     IBM Corporation - initial API and implementation
 -->
 
-<project name="wdt_imports">
+<project name="old_imports">
```

Comment:
Should it be olt_imports instead of old_imports?

Reply:
Good catch. Yes, that was a typo.